apollo-backend/src/main/java/io/logz/apollo/controllers/ScalingController.java

```diff
@@ -82,6 +90,24 @@ public void updateScalingFactor(int groupId, int scalingFactor, Req req) {
         group.setScalingStatus(Group.ScalingStatus.PENDING);
         groupDao.updateGroup(group);
 
+        String userEmail = req.token(Tokens._USER).toString();
```

Comment:
I am not very familiar with Rapidoid, but I see that another place is using **io.logz.apollo.auth.TokenConverter#convertTokenToUser**.
Is it enough to get by "_user" key? don't need to decrypt?

Reply:
Not exactly. TokenConverter is misleading, since it's taking the token directly from the HTTP Cookies header (only for web-sockets BTW), which is serialized, thus needs deserialization. In our case, Req object already have this deserialized and you can access the logged in User ID (which is email in our case) through the Standard mechanism.